projects/plugins/boost/app/lib/minify/Concatenate_CSS.php

```diff
@@ -192,10 +229,28 @@ public function do_items( $handles = false, $group = false ) {
 					$style_tag = "<link rel='stylesheet' id='$css_id' href='$href' type='text/css' media='$media' />";
 				}
 
+				/**
+				 * Filter the style loader HTML tag for page optimize.
+				 *
+				 * @param string $tag style loader tag
```

Comment:
To match with the filter usage:

```suggestion
				 * @param string $style_tag style loader tag
```